test/fuzzing/oss-fuzz-build.sh

```diff
@@ -1,3 +1,13 @@
 #/bin/bash -eu
+printf "package policy\nimport _ \"github.com/AdamKorcz/go-118-fuzz-build/testing\"\n" > $SRC/cilium/pkg/policy/registerfuzzdep.go
+go mod tidy && go mod vendor
+mv $SRC/cilium/pkg/policy/l4_filter_test.go $SRC/cilium/pkg/policy/l4_filter_test_fuzz.go
```

Comment:
Can you explain why we need to move these files around and where is this file being run? Can it be run locally by developers?

Maybe we should just move all fuzzing tests from  `l4_filter_test.go` to `l4_filter_test_fuzz.go` and not bother with moving the files around?

Reply:
the oss-fuzz-build.sh is meant to only be run inside the OSS-Fuzz build environment. In that environment, the fuzzers are built with go build which doesn't include `_test.go` files in its scope and they are therefore renamed. When run locally by developers with `go test -fuzz` then the renaming is not necessary.